WorkflowGuidelines.md

```diff
@@ -79,6 +79,32 @@
       let savedData = System.IO.File.ReadAllText saveFilePath.FullName
       ```
 
+    * Discarding generic exceptions:
+
+        Discarding generic exceptions is a bad practice because it can lead to unexpected behavior and bugs. It's better to handle the exception in some way, or at least log them to help with debugging if something unexpected happens.
+    
+        Example:
+        ```fsharp
+        let Func (zipFileInfo:FileInfo) =
```

Comment:
@parhamsaremi 2 things:
- Please respect our F# style guidelines.
- No need to use such a long name, `zipFile` is fine.

Reply:
Fixed the name.

For the guidelines I took a look and found out that I needed space after ":" and also had to specify the type for ignore. let me know if I've missed something.